### DISCUSSION TITLE
"The system cannot find the file specified"

### DISCUSSION THREAD

**Discussion by MiniRomer** [YM162/gulagcleaner#21] [Q&A]
Hi I am not great with tech, but I am trying to learn. When I run the **gulagcleaner [-r] [-s] [-n] [-h] [-v] <file name>...** in cmd I keep getting **The system cannot find the file specified.** Am I missing a step with installing it? Does the file have to be in a specific place? 

Thank you for the help

**Comment by YM162** [comment]
Hi! Lets see if we can make it work :)

If for example you have a file called "mynotes.pdf", you can run the code with "gulagcleaner mynotes.pdf". For this to work you have to open the terminal in the folder where "mynotes.pdf" is stored.

Alternatively, you can just give the full path to the file. For example, "gulagcleaner C:/Users/YM162/Desktop/mynotes.pdf"

If that doesn't work, tell me more about your OS, gulagcleaner version (get it with "gulagcleaner -v") and Python version ("python --version"), as it could be a bug in the code I haven't catched!

**Comment by MiniRomer** [comment]
![Screenshot 2024-06-03 153726](https://github.com/YM162/gulagcleaner/assets/87385609/9a7ceb62-6907-44d0-bd6f-51c7b6cbfba2)
Ok I think we are getting somewhere. I got this back I created another document, but it wasn't cleaned.

**Reply by MiniRomer** [reply]
I use windows 11, python 3.12 (64 bit) and I don't know where to find the github version.
